koku/masu/processor/aws/aws_report_parquet_summary_updater.py

```diff
@@ -57,7 +59,8 @@ def update_summary_tables(self, start_date, end_date, **kwargs):
         start_date, end_date = self._get_sql_inputs(start_date, end_date)
 
         with schema_context(self._schema):
-            self._handle_partitions(self._schema, UI_SUMMARY_TABLES, start_date, end_date)
+            partition_summary_tables = UI_SUMMARY_TABLES + (AWS_CUR_TABLE_MAP["ec2_compute_summary"],)
+            self._handle_partitions(self._schema, partition_summary_tables, start_date, end_date)
```

Comment:
Why are we adding this ec2 table here?

Reply:
This seems to be where we make sure partitions exist for the UI facing tables.

Reply:
ok, thats fine